### DISCUSSION TITLE
Preventing users to use "if" in a kata

### DISCUSSION THREAD

**Discussion by Daniel-Chen-1002** [codewars/codewars.com#2422] [Q&A]
How to ban users to use "if" in my created kata?

**Comment by DonaldKellett** [comment]
It's rarely a good idea to ban the use of certain language features in a Kata. While it may be possible to restrict them in some cases (such as using a regular expression to detect certain keywords), such techniques are rarely foolproof and many workarounds exist to circumvent these restrictions.

If you are new to authoring Kata on Codewars, you may wish to join our [Discord chat](https://discord.gg/mSwJWRvkHA) and post a link to your draft Kata in the #help-author channel, before you publish it to Beta. That way, you can receive feedback on your Kata before it is subject to the voting process (which may not always be in your favor).
